.github/workflows/ci.yaml

```diff
@@ -8,7 +8,7 @@ permissions:
   contents: read
   pull-requests: write
 env:
-  BUF_VERSION: "1.35.0"
+  BUF_VERSION: "1.50.1"
```

Comment:
Does this mean we are no longer testing against the actual minimum version we claim to support?

Reply:
Yes good catch, I updated the version to improve the examples. Changed this back and added a comment.